qemu/tests/cfg/blockdev_snapshot_stress.cfg

```diff
@@ -0,0 +1,16 @@
+- blockdev_snapshot_stress:
+    type = blockdev_snapshot_stress
+    virt_test_type = qemu
+    start_vm = yes
+    storage_type_default = "directory"
+    storage_pool = default
+    snapshot_tag = sn1
+    image_format_sn1 = qcow2
+    image_name_sn1 = images/sn1
+    device = "drive_image1"
+    base_tag = "image1"
+    rebase_mode = unsafe
```

Comment:
Do you need a variants to test  rebase_mode != unsafe ?

Reply:
No, rebase_mode = unsafe is just a workaround for a product bug.